docs/docs/experimentation-analysis/data-source-configuration.mdx

```diff
@@ -90,6 +90,16 @@ These queries are very simple and just need to return columns for each of the id
 SELECT user_id, device_id FROM logins
 ```
 
+##### Identifier Join Tables - Multiple identifiers
+
+For cases when you'd want to use multiple identifiers, it's recommended you use the `CONCAT` function to synthesize a single new identifier for joining.
+
+```
+SELECT CONCAT(company_id, user_id) as company_user
+```
+
+Then use `company_user` as an identifier throughout the app.
+
```

Comment:
This is not related to Identifier Join Tables.  Probably belongs under the `Identifier Types` section above instead.